api/envoy/extensions/load_balancing_policies/least_request/v3/least_request.proto

```diff
@@ -58,4 +59,9 @@ message LeastRequest {
 
   // Configuration for local zone aware load balancing or locality weighted load balancing.
   common.v3.LocalityLbConfig locality_lb_config = 4;
+
+  // Configuration for performing full scan on the list of hosts.
+  // If this configuration is set, when selecting the host a full scan on the list hosts will be
+  // used to select the one with least requests instead of using random choices.
+  google.protobuf.BoolValue full_scan_hosts = 5;
```

Comment:
maybe name it `enable_full_scan`. And it's necessary to tell if the `enable_full_span` is set to true, the `choice_count` will be ignored. I am not sure, but maybe this field and `choice_count` should be in a `oneof` (?). 

cc @lizan

Reply:
Changed to be enable_full_scan instead.